proto/remote_execution.proto

```diff
@@ -2266,3 +2269,22 @@ message StoredExecution {
   int32 status_code = 29;
   int32 exit_code = 30;
 }
+
+// Metadata associated with a firecracker VM.
+message VMMetadata {
+  // A UUID assigned to the VM upon creation which is preserved across
+  // snapshots.
+  string vm_id = 1;
+
+  message Creator {
+    // The ID of the invocation that created this snapshot.
+    string invocation_id = 1;
+
+    // The ID of the execution that created this snapshot.
+    string execution_id = 2;
+  }
+
+  // Who created the last snapshot that this VM was resumed from.
+  // If the VM was created from scratch, this field will be unset.
+  Creator last_snapshot_creator = 2;
```

Comment:
Would naming this something like `last_snapshot_user` be more clear? Especially if we are using this field in code for example and don't have the additional comments, I could see confusion around whether this is whatever created the snapshot from scratch

Reply:
hm I think `user` is inaccurate because this is who's creating the snapshot, not who's using it.

I could sort of see the confusion with the `Create()` method but can't think of a better alternative :shrug:

Reply:
actually wdyt about `snapshot_updated_by`?

Reply:
How about `snapshot_last_updated_by`? That is clearer to me than `last_creator` though!

Reply:
on second thought :sweat_smile: I realized snapshots are immutable so I'm not sure I like "updated" actually. I changed it to `last_executed_task` which I think is unambiguous - it's the last task that was executed on the VM (which should be the task that took the snapshot). wdyt?